src/posts/2020-10-14-wcag2.md

```diff
@@ -0,0 +1,96 @@
+- Information and structure must be separate from presentation.
+- Make it easy to distinguish foreground information from the background (for example good color contrast).
+
+2. Interfaces must be `operable (O)`. 
+
+- All functionality must be operable via keyboard.
+- All users must be able to control limits on their reading or interaction.
+- Users must be to avoid content that can cause seizures due to photosensitivity.
+- Provide mechanisms for users to find content, orientate themselves and navigate through it.
+- Help users avoid  mistakes and make it easier to correct mistakes when they occur.
+
+3. Content and controls must be `understandable (U)`.
+
+- Make text content readable and understandable.
+- Make the placement and functionality of content predictable.
+
+4. Content must be `robust (R)`.
+
+- Support compatibility with current and future user agents.
+- Ensure that content is accessible, or provide accessible alternatives.
+
+### Layers of guidance
+
+WCAG 2.0 provides various ways to help authors meet its standard called `layers of guidance`. They include the following :
+
+- Guidelines.
+- Success criteria.
+- Techniques.
+- Common failures.
+
+### Guidelines
+
+WCAG 2.0 guidelines are categorized into 3 [levels of conformance](https://www.w3.org/WAI/WCAG21/Understanding/conformance#levels) in order to meet the needs of different groups and situations. Under each principle, there is a [list of guidelines](https://www.w3.org/TR/WCAG20/#guidelines) that addresses the principles. 
+
+### Success criteria
+
+Under each guideline, there are success criteria that describe specifically what is to be done in order to conform to the standard.  Each success criteria is written as a statement that will be either true or false when specific web content is tested against it.  A convenient [list of the success criteria](https://www.w3.org/WAI/WCAG21/quickref/) can be found here.
+
+### Techniques
+
+The techniques document provides guidance for web authors and evaluators on meeting WCAG 2.0  success criteria. A convenient l[ist of the WCAG2.0 techniques](https://www.w3.org/TR/WCAG20-TECHS/) can be found here.
+
+### Common failures
+
+These are common mistakes that are considered failures of Success Criteria of WCAG 2.0. An [elaborate list of  WCAG2.0 common failures](https://www.w3.org/TR/WCAG20-TECHS/failures) can be found here.
+
+## What's new?
+
+WCAG 2.1 was published on June 2018. It was initiated with the aim of improving accessibility guidance for three major groups: users with cognitive or learning disabilities, users with low vision and users with disabilities on mobile devices. There are additional success criteria in WCAG 2.1 that are not in WCAG2.0. They are introduced in [What’s New in WCAG 2.1](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/). For  a  detailed  review of WCAG 2.1 , see  [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/)
```

Comment:
```suggestion
WCAG 2.1 was published on June 2018. It was initiated with the aim of improving accessibility guidance for three major groups: 

1. People with cognitive or learning disabilities, 
1. People with low vision, and 
1. People with disabilities on mobile devices. 

There are additional success criteria in WCAG 2.1 that are not in WCAG 2.0. They are introduced in [What’s New in WCAG 2.1](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/). 

For a detailed  review of WCAG 2.1 , see  [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/)
```

Broke out a list for easier reading. We also [prefer the term "person" to "user"](http://localhost:3000/content-style-guide/#user-or-person-).

Reply:
Thanks a lot that's very thoughtful. Forgive me for using the term "user".